src/EditorFeatures/CSharpTest/GenerateDefaultConstructors/GenerateDefaultConstructorsTests.cs

```diff
@@ -590,7 +591,7 @@ protected Program(SerializationInfo info, StreamingContext context) : base(info,
     {
     }
 }",
-index: 3,
+index: 4,
```

Comment:
Because GenrateDefaultConstructors now offers to generate the no arg constructor, the indices increased here.